palace/drivers/basesolver.cpp

```diff
@@ -565,6 +566,44 @@ void BaseSolver::PostprocessFields(const PostOperator &postop, int step, double
   Mpi::Barrier();
 }
 
+void BaseSolver::PostprocessErrorIndicators(const std::string &name, int step, double time,
+                                            const std::array<double, 5> &data) const
+{
+  if (post_dir.length() == 0)
+  {
+    return;
+  }
+
+  // Write the indicator statistics
+  if (root)
+  {
+    std::string path = post_dir + "error-indicators.csv";
+    auto output = OutputFile(path, (step > 0));
+    if (step == 0)
+    {
+      // clang-format off
+      output.print("{:>{}s},{:>{}s},{:>{}s},{:>{}s},{:>{}s},{:>{}s}\n",
+                   name, table.w1,
+                   "Sum Rel.", table.w,
```

Comment:
For the column headings, I propose removing the `"Elem"` and the `"Rel"`. Thus, it will just be: `"Sum", "Min", "Max", "Mean", "Normalization"`.

The meaning of the quantities as sum/min/max/mean over all elements of the relative quantity already divided by the normalization can be clarified in the documentation (to come in a later PR).